csvcubed/csvcubed/cli/inspect/inspect.py

```diff
@@ -35,7 +35,7 @@ def inspect(csvw_metadata_json_path: Path) -> None:
     metadata_processor = MetadataProcessor(csvw_metadata_json_path)
     csvw_metadata_rdf_graph = metadata_processor.load_json_ld_to_rdflib_graph()
 
-    assert csvw_metadata_rdf_graph is not None
+    assert csvw_metadata_rdf_graph is not None, "The rdf graph cannot be None."
```

Comment:
Can we avoid using asserts and raise exceptions instead? Assertions are turned off when running python in an optimised mode.

https://docs.python.org/3/using/cmdline.html#cmdoption-O

Reply:
but it's fine to use asserts in tests of course